src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v6/InternalUserV6.java

```diff
@@ -79,6 +79,7 @@ public String getHash() {
         public void setHash(String hash) {
             this.hash = hash;
         }
+        public void setPassword(String password){}
```

Comment:
Please add comments explaining why the empty setter for password is necessary. Follow code style.